cmake/Modules/ObsHelpers_macOS.cmake

```diff
@@ -354,13 +354,21 @@ function(setup_obs_bundle target)
     set(_CODESIGN_ENTITLEMENTS \"${CMAKE_SOURCE_DIR}/cmake/bundle/macOS\")"
     COMPONENT obs_resources)
 
-  if(ENABLE_SPARKLE_UPDATER)
+  if(DEFINED SPARKLE_APPCAST_URL AND DEFINED SPARKLE_PUBLIC_KEY)
```

Comment:
Remove the `DEFINED` part, because this will enable Sparkle even if the values are empty strings.

`0`, `OFF`, `FALSE`, ``, and `<NAME>-NOTFOUND` are all treated as `false` in CMake.

Reply:
I guess we should fix that in master as well then 😅